Packs/CommonScripts/Scripts/IsInCidrRanges/IsInCidrRanges.py

```diff
@@ -1,19 +1,30 @@
-import demistomock as demisto
-from CommonServerPython import *
+import ipaddress
 
-from netaddr import IPAddress, IPNetwork
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
 
 
 def main():
-    ip_address = demisto.args()['left']
+    """ Check if given IP (or IPs) address is part of a given CIDR (or a list of CIDRs).
+
+    Args:
+        ip_addresses (str): A list of IPs/IPv6s
+        cidr_range_list (str): A list of CIDRs to be checked against.
+
+    Returns:
+        bool: True if given IP is part of given CIDR range.
+    """
+
+    ip_addresses = argToList(demisto.args()['left'])
     cidr_range_list = argToList(demisto.args()['right'])
 
-    for cidr in cidr_range_list:
-        if IPAddress(ip_address) in IPNetwork(cidr):
-            demisto.results(True)
-            return
+    for ip in ip_addresses:
+        for cidr in cidr_range_list:
+            if ipaddress.ip_address(ip) in ipaddress.ip_network(cidr):
+                demisto.results(True)
+                return
 
-    demisto.results(False)
+        demisto.results(False)
```

Comment:
The result should be false only after we checked all IPs, isn't it?
So the previous indentation should be restored.

Reply:
I fixed the code, it gets two lists and checks if the values in list A are contained in at least one of the values in list B
![image](https://user-images.githubusercontent.com/24267931/165498785-3d6ddcd4-a499-4498-b847-eb49d45f6582.png)